core/input/generisdata_tech.prn

```diff
@@ -197,14 +197,13 @@ Explanations:
  
 Electrolysis - elh2
 learning parameterization:
-Assume that 20 GW(el) electrolysis installed globally at 1200 EUR/GW(el) in 2025 
-(Ueckerdt et al. (2021), https://doi.org/10.1038/s41558-021-01032-7, Fig. S3,
- Odenweller et al. (2022), https://doi.org/10.1038/s41560-022-01097-4, Fig. 1d ).
-
-20 GW(el) * 0.65 (elh2 efficiency) * 1e-3 ~ 0.013 TW(H2) = ccap0
-1200 EUR/GW(el) / 0.65 (elh2 efficiency) * 1.1 (EUR to USD, 2015) ~ 2000 USD/kW(H2) = inco0
-Assume floor cost: 350 EUR/GW(el)
-350 EUR/GW(el) / 0.75 (long-term elh2 efficiency) * 1.1 (EUR to USD, 2015) ~ 500 USD/kW(H2) -> incolearn = 1500
+Assume that 10 GW(el) electrolysis installed globally and CAPEX of 900 EUR/kW(el) in 2025 
+(Odenweller et al. (2022), https://doi.org/10.1038/s41560-022-01097-4,
+Ueckerdt et al. (2021), https://doi.org/10.1038/s41558-021-01032-7, Fig. S3).
+10 GW(el) * 0.65 (elh2 efficiency) * 1e-3 ~ 0.0065 TW(H2) = ccap0
+900 EUR/kW(el) / 0.65 (elh2 efficiency) * 1.1 (EUR to USD, 2015) ~ 1500 USD/kW(H2) = inco0
```

Comment:
Falko suggested 800 EUR2020/kW(el), didn't he?

Reply:
Yeah, in the current implementation it still is a bit lower than the initialization value anyways in 2025. Gonna check when looking at the results.